src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -836,9 +882,8 @@ public static IEnumerable<object[]> SendAsync_SendSameRequestMultipleTimesDirect
             }
         }
 
-        [OuterLoop("Uses external servers")]
+        [OuterLoop("Uses external servers", TestPlatforms.Any & ~TestPlatforms.Browser)]
```

Comment:
I would suggest adding a `PlatformDetection.IsLocalEchoServerAvailable`, which can be true for browser. Then we can condition `OuterLoop` on `!PlatformDetection.IsLocalEchoServerAvailable`

Reply:
Good idea to refactor this, will do, thanks. It's not bool but `TestPlatforms` enum.